src/main/resources/application.conf

```diff
@@ -31,6 +31,10 @@ rokku {
             verifyToken {
                 checkRealmUrl = ${?KEYCLOAK_CHECK_REALM_URL}
                 issuerForList = ${?KEYCLOAK_CHECK_ISSUER_FOR_LIST}
+                NPAClaim = "NPA"
+                NPAClaim = ${?KEYCLOAK_NPA_CLAIM_KEY}
+                NPAClaimContains = "true"
```

Comment:
Is the value needed? Maybe to know if it is NPA we can check if the claim key exist?
If the value is expected to be plural what other values we expect?